kubemarine/resources/configurations/globals.yaml

```diff
@@ -331,7 +351,7 @@ compatibility_map:
       - os_family: 'rhel8'
         versions:
           - '8.6'
-          - '8.7'
+          - '8.8'
```

Comment:
Probably add 8.8, not replace already supported 8.7?

Reply:
add 8.7 version